### PR TITLE
Add About section to the website

### PR DESCRIPTION
**Pull request by Nerocodes**
Related [issue](https://github.com/react-bootstrap/react-bootstrap/issues/4624)

### CODE REVIEW

**Comment by mxschmitt** [comment]
cc @bpas247 what do you think? For an initial version in my mind already very good.

File: www/src/pages/about.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+We currently aren't accepting any funding what so ever, but that could change 😉
+
+### Get Involved
+Get involved with React-Bootstrap [by opening an issue](https://github.com/react-bootstrap/react-bootstrap/issues/new) or submitting a pull request.
+See our [contributing guidelines](https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md) here.
+
+### External Links
+- [Bootstrap](https://getbootstrap.com/)
+- [React](https://reactjs.org/)
+- [React-router-bootstrap](https://github.com/react-bootstrap/react-router-bootstrap)
```
Comment (bpas247) [inline www/src/pages/about.mdx]:
```suggestion
- [React Router Bootstrap](https://github.com/react-bootstrap/react-router-bootstrap)
```
hmm, not entirely sure if it should be `React Router Bootstrap` or `react-router-bootstrap`. The former seems to make more sense for consistency purposes

File: www/src/pages/about.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+### Team
+React-Bootstrap is maintained by a [team of developers](https://github.com/orgs/react-bootstrap/people) on Github. We have a growing team
+and if you are interested in re-building the most popular front-end framework with React we would love to hear from you.
+
+### Contributors
+We welcome community support with both pull requests and reporting bugs. Please don't hesitate to jump in.
+Join our growing list of [contributors](https://github.com/react-bootstrap/react-bootstrap/graphs/contributors).
+
+### Sponsorships
+We currently aren't accepting any funding what so ever, but that could change 😉
```
Comment (bpas247) [inline www/src/pages/about.mdx]: I'm not sure if we should even have this blurb? I haven't really discussed with the other maintainers as to their thoughts on funding and sponsorships yet, we could just omit this section for now.

File: www/src/pages/about.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+### Get Involved
+Get involved with React-Bootstrap [by opening an issue](https://github.com/react-bootstrap/react-bootstrap/issues/new) or submitting a pull request.
+See our [contributing guidelines](https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md) here.
+
+### External Links
+- [Bootstrap](https://getbootstrap.com/)
+- [React](https://reactjs.org/)
+- [React-router-bootstrap](https://github.com/react-bootstrap/react-router-bootstrap)
+- [Awesome React Bootstrap Components](https://github.com/Hermanya/awesome-react-bootstrap-components)
+- React-Bootstrap [CodeSandbox examples](https://codesandbox.io/s/github/react-bootstrap/code-sandbox-examples/tree/master/basic)
```
Comment (bpas247) [inline www/src/pages/about.mdx]:
```suggestion
- [React Bootstrap CodeSandbox examples](https://codesandbox.io/s/github/react-bootstrap/code-sandbox-examples/tree/master/basic)
```

File: www/src/pages/about.mdx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# About
+<p className="lead">
+	Get to know more about the team maintaining React-Bootstrap. Learn a
```
Comment (bpas247) [inline www/src/pages/about.mdx]:
```suggestion
	Get to know more about the team maintaining React Bootstrap. Learn a 
```

File: www/src/pages/about.mdx [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# About
+<p className="lead">
+	Get to know more about the team maintaining React-Bootstrap. Learn a 
+	little history of how, why and when the project started and how
+	you can be a part of it.
+</p>
+
+### Team
+React-Bootstrap is maintained by a [team of developers](https://github.com/orgs/react-bootstrap/people) on Github. We have a growing team
```
Comment (bpas247) [inline www/src/pages/about.mdx]:
```suggestion
React Bootstrap is maintained by a [team of developers](https://github.com/orgs/react-bootstrap/people) on Github. We have a growing team
```

File: www/src/pages/about.mdx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+and if you are interested in re-building the most popular front-end framework with React we would love to hear from you.
+
+### Contributors
+We welcome community support with both pull requests and reporting bugs. Please don't hesitate to jump in.
```
Comment (bpas247) [inline www/src/pages/about.mdx]:
```suggestion
We welcome community support with both feature requests and bug reporting. Please don't hesitate to jump in.
```

File: www/src/pages/about.mdx (hunk 1 above)
Comment (Nerocodes) [reply www/src/pages/about.mdx]: Alright! Changes made.

File: www/src/pages/about.mdx (hunk 2 above)
Comment (Nerocodes) [reply www/src/pages/about.mdx]: Done

File: www/src/pages/about.mdx (hunk 4 above)
Comment (Nerocodes) [reply www/src/pages/about.mdx]: Done

File: www/src/pages/about.mdx (hunk 5 above)
Comment (Nerocodes) [reply www/src/pages/about.mdx]: Done

File: www/src/pages/about.mdx (hunk 6 above)
Comment (Nerocodes) [reply www/src/pages/about.mdx]: Done

**Comment by bpas247** [comment]
@Nerocodes thanks so much for contributing 😃 great work!

**Pull request merged**